src/org/omegat/filters2/html2/FilterVisitor.java

```diff
@@ -245,6 +238,21 @@ public void visitTag(Tag tag) {
         }
     }
 
+    private boolean isTranslateAttribute(Tag tag) {
+        if (!options.getTranslateValue() && !options.getTranslateButtonValue()) {
+            return false;
+        }
+
+        // an input element
+        // and we translate all input elements
+        // or we translate submit/button/reset elements ...
+        // and it is a submit/button/reset element.
+        // then translate the value
+        return "submit".equalsIgnoreCase(tag.getAttribute("type"))
```

Comment:
There are multiples of `tag.getAttribute("type")` Can we introduce local variable and use it?